yesod-form/Yesod/Form/I18n/Chinese.hs

```diff
@@ -0,0 +1,26 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Yesod.Form.I18n.Chinese where
+
+import Yesod.Form.Types (FormMessage (..))
+import Data.Monoid (mappend)
+import Data.Text (Text)
+
+chineseFormMessage :: FormMessage -> Text
+chineseFormMessage (MsgInvalidInteger t) = "无效的整数: " `Data.Monoid.mappend` t
```

Comment:
Just a style issue: To be consistent with the rest of the code, you can use `mappend` here also instead of `Data.Monoid.mappend`.